src/Components/GenreSelector.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export class GenreSelector extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selected: parseInt(props.selectedGenre),
+            genreList: props.genreList
+        };
+    }
+
+    selectGenre(genre){
+        let state = {...genre};
+        state.selected = genre.id;
+        this.setState(state);
+        this.props.onSelectGenre(genre)
+    }
+
+    render() {
+        return (
+            <div className='genreSelectContainer'>
+                <li>
+                    {this.state.genreList.map(
```

Comment:
nit: keep in mind, that sometimes you might rendering empty lists, so consider other scenarios in the future such as empty list

Reply:
In this case, an empty list should be boarded in a different way?